### PR TITLE
[APP-74] add defaultSorting option (2)

### PR DESCRIPTION
**Pull request by alexstrat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,35 @@ const { openProcessManager } = require('electron-process-manager');
...
+| Idle Wake Ups /s   | 'cpu.idleWakeupsPerSecond' |
+| WebContents Id     | 'webContents.0.id'         |
+| WebContents Type   | 'webContents.0.type'       |
+| WebContents URL    | 'webContents.0.URL'        |
+
+example:
+```js
+const { openProcessManager } = require('electron-process-manager');
+
+openProcessManager({ how: 'descending', path: 'cpu.percentCPUUsage' });
```
Comment (alexstrat) [inline README.md]: `openProcessManager({defaultSorting: { how: 'descending', path: 'cpu.percentCPUUsage' }});`?

**Comment by alexstrat** [comment]
@guillaumearm added `defaultSorting` to example in 7931766 but does not look like having an impact when I'm testing.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,35 @@ const { openProcessManager } = require('electron-process-manager');
...
+
+| Field name         | path                       |
+|--------------------|----------------------------|
+| Pid                | 'pid'                      |
+| WebContents Domain | 'webContents.0.URLDomain'  |
+| Process Type       | 'webContents.0.type'       |
+| Private Memory     | 'memory.privateBytes'      |
+| Shared Memory      | 'memory.sharedBytes'       |
+| Working Set Size   | 'memory.workingSetSize'    |
+| % CPU              | 'cpu.percentCPUUSage'      |
```
Comment (guillaumearm) [inline README.md]: typo here: `cpu.percentCPUUsage`

**Comment by guillaumearm** [comment]
@alexstrat: The cause is due to a typo in documentation 🙂.
I will fix that soon

**Comment by guillaumearm** [comment]
[electron-process-manager@0.6.0-rc0](https://www.npmjs.com/package/electron-process-manager/v/0.6.0-rc0) released on npm

**Pull request merged**